problem_builder/mentoring.py

```diff
@@ -236,7 +236,9 @@ def student_view(self, context):
 
         for child_id in self.children:
             child = self.runtime.get_block(child_id)
-            if not isinstance(child, MentoringMessageBlock):
+            if child is None:  # child should not be None but it can happen due to bugs or permission issues
+                child_content += u"<p>[{}]</p>".format(self._(u"Error: Unable to load child component."))
+            elif not isinstance(child, MentoringMessageBlock):
```

Comment:
Also, in this case, if permissions are the problem, I would consider it a "configuration error" - the block is not designed to handle variable children, so if an author has configured it that way, they're using it incorrectly. Thankfully though that shouldn't be an issue in theory since there is no mechanism to do that AFAIK.